tests/integration/ceph_mgr_test.go

```diff
@@ -113,9 +113,26 @@ func (s *CephMgrSuite) TearDownSuite() {
 	s.installer.UninstallRook()
 }
 
-func (s *CephMgrSuite) execute(command []string) (error, string) {
-	orchCommand := append([]string{"orch"}, command...)
-	return s.installer.Execute("ceph", orchCommand, s.namespace)
+func (s *CephMgrSuite) execute(command []string, maxRetries int) (error, string) {
```

Comment:
convention in golang is for `error` to always be the last return value
```suggestion
func (s *CephMgrSuite) execute(command []string, maxRetries int) (string, error) {
```